### PR TITLE
Update prometheus dependencies to latest

### PR DESCRIPTION
**Pull request by vjsamuel**

### CODE REVIEW

**Comment by vjsamuel** [comment]
@ruflin i have done the needful

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -213,7 +213,7 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Docker and Kubernetes modules are now GA, instead of Beta. {pull}6105[6105]
 - Add pct calculated fields for Pod and container CPU and memory usages. {pull}6158[6158]
 - Add statefulset support to Kubernetes module. {pull}6236[6236]
-
+- Update prometheus dependencies to latest {pull}6333[6333]
```
Comment (exekias) [inline CHANGELOG.asciidoc]: there is a missing new line here

**Comment by exekias** [comment]
ok to test

**Comment by ruflin** [comment]
@vjsamuel This needs a `make update`.

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (ruflin) [reply CHANGELOG.asciidoc]: I added one :-)

**Comment by vjsamuel** [comment]
@exekias @ruflin is there anything blocking this?

**Comment by ruflin** [comment]
@vjsamuel Was waiting on me being at work :-)

**Pull request merged**